src/pages/Compare/utils.tsx

```diff
@@ -24,7 +24,7 @@ const mergeChanges = (
 
 // This is a map of an old vs. new file comparison, giving preference to the
 // "forward" file, i.e. the newer file.
-export class ForwardComparisonMap {
+export default class ForwardComparisonMap {
```

Comment:
Is there any reason besides the mocking problem to make this a default export? If so, I suggest changing the module name to `forwardComparisonMap.js` to avoid confusion.

Reply:
It was just because of the mocking. I'll change it back